### PR TITLE
pinephone: GSM audio enabling

### PR DESCRIPTION
**Pull request by matviizorin**
The initial configuration supports only one route:
    
    AIF1 (Advanced Peripheral Bus) <--> Line Out (Loudspeaker)
    
Add 4 additional initial routes:
    
    AIF1 <--> Mic1 (Internal microphone)
    AIF1 <--> AIF2 (Baseband)
    AIF2 <--> DAC (Digital-to-analog converter)
    AIF2 <--> Mic1

### CODE REVIEW

File: pinephone/device.mk [hunk 1]
```diff
@@ -42,7 +42,7 @@ PRODUCT_COPY_FILES += \
     $(LOCAL_PATH)/modem.pinephone.rc:$(TARGET_COPY_OUT_VENDOR)/etc/init/modem.pinephone.rc \
 
 PRODUCT_COPY_FILES += \
-    device/glodroid/pinephone/audio.pinephone.xml:$(TARGET_COPY_OUT_VENDOR)/etc/audio.pinephone.xml \
+    device/glodroid/pinephone/audio/audio.pinephone.xml:$(TARGET_COPY_OUT_VENDOR)/etc/audio.pinephone.xml \
```
Comment (rsglobal) [inline pinephone/device.mk]: Hello. Thanks for this series. I'm just wondering what is the rationale for this change?

Comment (matviizorin) [reply pinephone/device.mk]: Maybe some 'audio_policy_configuration' will be added in the future, so it is better to keep it in a separate directory.

Comment (rsglobal) [reply pinephone/device.mk]: Oh, i see, thanks. Please drop this patch from these series and apply it to series with `audio_policy_configuration` when ready.

Comment (matviizorin) [reply pinephone/device.mk]: Done

**Comment by rsglobal** [comment]
Thank you for this series. Looks good. Merging...
